lib/src/clojure_lsp/feature/hover.clj

```diff
@@ -123,7 +123,12 @@
                             (str join-char signatures)))
         markdown? (some #{"markdown"} content-formats)
         doc-line (find-docstring db markdown? uri doc 0)
-        clojuredocs (f.clojuredocs/find-hover-docs-for sym-name sym-ns db*)
+        clojuredocs (or (f.clojuredocs/find-hover-docs-for sym-name sym-ns db*)
```

Comment:
Can we add a test for that? otherwise if we change this behavior we wouldn't know we introduced a regression.
A unit test should be enough on hover_test.clj